robocorp-code/src/robocorp_code/playwright/__playwright__main.py

```diff
@@ -114,6 +118,7 @@ def in_thread():
             # -- note: this may be particular to my machine (fabioz), but it
             # may also be related to VSCode + Windows 11 + Windows Defender + python
             _stdin_write(process, b"\n")
+            _stdin_write(process, b"\n")
```

Comment:
Did this actually make a difference? The first call to `_stdin_write` should close `process.stdin`, so, in theory this will never do anything...